docs/ui/AdviceCard.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { ReactNode } from 'react';
+
+import { Card, Headline, Text } from '@marigold/components';
+
+import { cn } from '../../packages/system/dist';
+
+interface Props {
+  description: string;
+  type: 'DO' | "DON'T";
```

Comment:
I think it's a bad idea to write it uppercase and with a ' because its more error-prone, I would think

Reply:
and you can also change type to variant?